build.gradle.kts

```diff
@@ -50,16 +50,16 @@ allprojects {
     // and suggest an upgrade. The only exception currently are those defined
     // in buildSrc, most likley because the variables are used in plugins as well
     // as dependencies. e.g. KOTLIN_VERSION
-    extra["sb.version"] = "2.6.7"
+    extra["sb.version"] = "2.6.12"
     val springBootVersion = extra["sb.version"] as String
 
     dependencyRecommendations {
         mavenBom(mapOf("module" to "org.jetbrains.kotlin:kotlin-bom:${Versions.KOTLIN_VERSION}"))
-        mavenBom(mapOf("module" to "org.springframework:spring-framework-bom:5.3.18"))
+        mavenBom(mapOf("module" to "org.springframework:spring-framework-bom:5.3.23"))
```

Comment:
Changes to `spring-framework-bom`, `spring-security-bom`, and `jackson-bom` align with the versions used in SB 2.6.12.